src/compiler/utilities.ts

```diff
@@ -1134,6 +1140,48 @@ module ts {
     }
 
     // @internal
+    export function nodeStartsNewLexicalEnvironment(n: Node): boolean {

```

Comment:
What does this mean? Would it be correct to call this nodeStartsNewFunctionScope?